pandas/tests/resample/test_base.py

```diff
@@ -215,13 +214,7 @@ def test_resample_loffset_arg_type(frame, create_index, arg):
     if isinstance(arg, list):
         expected.columns = pd.MultiIndex.from_tuples([("value", "mean")])
 
-    # GH 13022, 7687 - TODO: fix resample w/ TimedeltaIndex
-    if isinstance(expected.index, TimedeltaIndex):
-        msg = "DataFrame are different"
-        with pytest.raises(AssertionError, match=msg):
-            tm.assert_frame_equal(result_agg, expected)
```

Comment:
so this is testing incorrect behavior?  thats pretty weird

Reply:
@jbrockmendel Yeah, that was the case. That was testing an incorrect behavior... :confused: And this was the case for two reasons:
* `loffset` support was missing in `.resample` when dealing with timedelta. (#7687)
* an extra bin (fixed with this PR) was present each time (#13022, #30353)

Here an example of the values when setting a breakpoint with PyCharm line 222 in `pandas/tests/resample/test_base.py` on **master**:

The expected result (with `loffset` and without extra bin):
![expected](https://user-images.githubusercontent.com/3902859/79496789-efe38780-8026-11ea-8f79-4594740db10c.png)

The actual result (without `loffset` and with an extra bin):
![result_agg](https://user-images.githubusercontent.com/3902859/79496791-f07c1e00-8026-11ea-8b44-6501c54be0ec.png)

I think that was the reason of the presence of this TODO:
https://github.com/pandas-dev/pandas/blob/3d4f9dc19d784526f71a197bfb6e36b0409e0760/pandas/tests/resample/test_base.py#L218